src/h5web/visualizations/shared/TooltipMesh.tsx

```diff
@@ -18,9 +25,14 @@ interface Props {
 function TooltipMesh(props: Props): ReactElement {
   const { formatIndex, formatValue, guides } = props;
 
+  const { abscissa, ordinate } = useContext(AxisSystemContext);
   const { camera, size } = useThree();
   const { width, height } = size;
 
+  // Scales to compute data coordinates from unprojected mesh coordinates
+  const abscissaScale = getAxisScale(abscissa, width);
+  const ordinateScale = getAxisScale(ordinate, height);
```

Comment:
The removal of the hooks requires retrieving the axis info objects. I think it actually makes things clearer. Also, we were already retrieving the size of the canvas anyway.